data/api/client-server/threads_list.yaml

```diff
@@ -32,8 +32,7 @@ paths:
       x-addedInMatrixVersion: "1.4"
       summary: Retrieve a list of threads in a room, with optional filters.
       description: |-
-        Paginates over the thread roots in a room, ordered by the `latest_event` of each thread root
-        in its bundle.
+        Paginates over the thread roots in a room.
```

Comment:
The description appears to have lost some context here?

Reply:
It seemed redundant to specify the sort order here as well as in the response.

Reply:
it adds a bit of clarity without having to scroll past a whole request schema, though not all that bothered.

The `summary` and `description` should be switched if we are losing the text though: it's more awkward if the `summary` is longer than the `description`.

Reply:
Have updated a bit more, with reference to https://github.com/matrix-org/matrix-spec/blob/main/meta/documentation_style.rst#openapi.

If you still don't like it, am happy to put it back how it was.